packages/notification-center-vue/vite.config.js

```diff
@@ -35,7 +35,7 @@ export default defineConfig({
       fileName: (format) => `index.${format}.js`,
     },
     rollupOptions: {
-      external: ['vue'],
+      external: ['vue', 'react', 'react-dom'],
```

Comment:
The `react, react-dom` are declared as the peer dependencies in the notification center package, meaning the package that will use it should have these as dev dependencies. Here we tell Rollup to include them in the `notification-center-vue` package.

Reply:
I guess that this solves the problem but I find weird that the Vue implementation has React dependencies. Might we be doing anything wrong somehow? 🤔

Reply:
Yes, it's intended. The `notification-center` package has defined `react, react-dom` as the `peer` dependencies, and this package `notification-center-vue` has a regular dependency on the `notification-center`, meaning it should fulfill that deps requirements.